cores/aliens/cfg/macros.def

```diff
@@ -31,7 +31,7 @@ JTFRAME_DIPBASE=8
 # JTFRAME_IOCTL_RD=19472
 JTFRAME_INPUT_RECORD
 
-[mist|sidi]
+[mist|sidi|demist_*|mist_*]
```

Comment:
I think `[*mist*|sidi]` would work too

Reply:
I think *mist* is not good as it will include mister as well.

Reply:
Right!